home/modules/contribute/pages/test-site.adoc

```diff
@@ -17,15 +17,16 @@ If you left some remote repositories in your local playbook file, make sure you
 Keep the PAT somewhere where you can copy it again later.
 +
 Make sure you also authorize your token to work with SAML single sign-on. 
-. Open a Git Bash window. 
+. Open a terminal window. 
 +
+NOTE: If you're on Windows, you must do the following steps with Git Bash, not any other terminal program.
 You should have automatically installed Git Bash when you installed Git on your computer. 
 . Paste the following command: 
 +
 [source, console]
 ----
 git config --global credential.helper store && \
-git ls-remote -h https://github.com/couchbase/couchbase-operator > /dev/null && \
+git ls-remote -h <GITHUB_REPO_URL> > /dev/null && \
```

Comment:
is that what we agreed for placeholder variables?
In the case of shell commands, `$GITHUB_REPO_URL` could also work

You could then prefix with:

```
# Set this variable to the appropriate URL for the repository in question:
export GITHUB_REPO_URL=https://github.com/couchbase/couchbase-operator
```

Reply:
Good point. Correcting...